src/main/scala/cromwell/engine/backend/BackendCall.scala

```diff
@@ -170,8 +170,12 @@ trait BackendCall {
   def hash(implicit ec: ExecutionContext): Future[ExecutionHash] = {
     // If a Docker image is defined in the task's runtime attributes, return a `Future[Option[String]]` of the Docker
     // hash string, otherwise return a `Future.successful` of `None`.
-    val eventualDockerHash = call.task.runtimeAttributes.docker map {
-      backend.dockerHashClient.getDockerHash(_) map { dh => Option(dh.hashString) } } getOrElse Future.successful(None)
+    val eventualDockerHash = call.task.runtimeAttributes.docker map { dockerImage =>
+      if (workflowDescriptor.lookupDockerHash)
+        backend.dockerHashClient.getDockerHash(dockerImage) map { dh => Option(dh.hashString) }
+      else
+        Future.successful(Some(dockerImage))
```

Comment:
> should we hash the image name rather than returning the name as the hash?

is there any benefit to hashing the string?

> also should always use Option rather than Some to lift raw values into Option.

Yes, indeed, I'll change